src/boss/itachi_boss.java

```diff
@@ -1,2 +1,39 @@
+import nl.saxion.app.SaxionApp;
+
 public class itachi_boss {
```

Comment:
maybe its a better idea for us to create classes based on the previous one created "Entity" . and store all the basic information that we gonna use for all of characters like : name , x , y , images , etc ....